parser/model/model.go

```diff
@@ -430,6 +430,8 @@ type TableInfo struct {
 	MaxShardRowIDBits uint64 `json:"max_shard_row_id_bits"`
 	// AutoRandomBits is used to set the bit number to shard automatically when PKIsHandle.
 	AutoRandomBits uint64 `json:"auto_random_bits"`
+	// IntPKRangeBits represents the bit number of the int primary key that will be used by TiDB.
+	IntPKRangeBits uint64 `json:"int_pk_range_bits"`
```

Comment:
Does it be used besides autoRandom? Maybe `AutoRandomRangeBits` is better?

Reply:
Updated.